lewis/examples/modbus_device/__init__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# *********************************************************************
+# lewis - a library for creating hardware device simulators
+# Copyright (C) 2016 European Spallation Source ERIC
```

Comment:
Doing some research on this:
* Copyright notices are generally optional and purely educational (no bearing on legal protection)
* The year should be the first year of publication of a work
* Optionally, you may update the year (or list years) for any year with significant changes made
* GNU recommends: `2008, 2009, 2010` or `2008-2010` format
For purposes of software, a "work" is the entire software package as a whole, not individual files.

Long story short: 
We can either leave it `2016` everywhere and just not worry about it, or, if we want to update, we should search replace the entire source folder to `2016-2017` or similar. Best to do it in one go as a PR if we go that route, I think.